### DISCUSSION TITLE
Orphan

### DISCUSSION THREAD

**Discussion by ypool-xiaoyaozi** [filecoin-project/lotus#6105] [Ideas] [answered]
https://www.filscout.com/  With the opening of isolated block query, too many nodes lose too much revenue. Can we apply Ethereum TERT block model to design

**Comment by ypool-xiaoyaozi** [comment]
Design using ethereum uncle block model

**Comment by ypool-xiaoyaozi** [comment]
I think this can save a bit of the miner's loss.

**Comment by SgtCoin** [comment, marked as answer]
Archiving for future reference due to age.
